src/stp/new_tactics/Halt.cpp

```diff
@@ -19,7 +19,7 @@ void Halt::onTerminate() noexcept {}
 
 std::optional<StpInfo> Halt::calculateInfoForSkill(StpInfo const &info) noexcept {
     StpInfo skillInfo = info;
-    skillInfo.setAngle(0);
+    skillInfo.setAngle(0.00001);
```

Comment:
Huh? How can we accurately move 0.00001 degrees?

Reply:
This indeed looks weird and if you write such expressions, make sure you use the E notation. 1e-5 is more clear to me than 0.00001